src/python/turicreate/test/test_style_transfer.py

```diff
@@ -161,7 +161,7 @@ def test_stylize_fail(self):
             with self.assertRaises(_ToolkitError):
                 model.stylize(self.content_sf[0:1], style=style)
 
-    @unittest.skip("Fix in a Later PR, known failure")
+    @unittest.skip("Fix before 6.0 release candidate, known failure")
```

Comment:
If you rebase this file, you can do `@pytest.mark.xfail(IS_PRE_6_0_RC, reason='Requires MXNet')`

Reply:
Thanks for the heads up! I don't want to risk rebasing right now. I added TODO's for the next PR after this one goes in!